pandas/io/parsers.py

```diff
@@ -841,6 +841,16 @@ def _clean_options(self, options, engine):
                                       encoding=encoding)
                 engine = 'python'
 
+        quotechar = options['quotechar']
+        if (quotechar is not None and
+                isinstance(quotechar, (str, compat.text_type, bytes))):
+            if (len(quotechar) == 1 and ord(quotechar) > 127 and
+                    engine not in ('python', 'python-fwf')):
+                fallback_reason = ("the quotechar is > 1 char long, and "
+                                   "the 'c' engine does not support such "
+                                   "quotechars".format(encoding=encoding))
```

Comment:
it's nice to the reason is more clearer, like "quotechar must be a single byte character, which ord(quotechar) <= 127". pls correct my English:)